app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTask.java

```diff
@@ -170,7 +167,7 @@ public char getPriority() {
             if (ret.length() == 1) {
                 priority = ret.charAt(0);
             } else {
-                priority = 0;
+                priority = '~'; // No priority == lowest priority
```

Comment:
`~` has a 'high' value per ascii. This ensures that tasks with no priority have low priority